crates/router/src/core/user/dashboard_metadata.rs

```diff
@@ -3,7 +3,13 @@ use diesel_models::{
     enums::DashboardMetadata as DBEnum, user::dashboard_metadata::DashboardMetadata,
 };
 use error_stack::ResultExt;
+#[cfg(feature = "email")]
+use router_env::logger;
 
+#[cfg(feature = "email")]
+use crate::services::email::types as email_types;
+#[cfg(feature = "email")]
+use crate::types::domain::{self};
```

Comment:
I don't think you need self here.

Reply:
It is required As it is throwing error when I'm removing it.